data-model/virtualcollections/virtualcollections.json

```diff
@@ -1 +1,99 @@
-
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
+  "type": "object",
+  "properties": {
+    "ods:id": {
+      "type": "string",
+      "pattern": "^https:\\/\\/hdl\\.handle\\.net\\/20\\.5000\\.1025\\/.{3}-.{3}-.{3}$",
+      "examples": ["https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX"]
+    },
+    "ods:type": {
+      "type": "string",
+      "description": "The FDO type of the object"
+    },
+    "ods:version": {
+      "type": "integer",
+      "description": "The version of the object."
+    },
+    "dcterms:creator": {
+      "type": "object",
+      "properties": {
+        "ods:type": {
+          "type": "string"
+        },
+        "foaf:name": {
+          "type": "string"
+        }
+      },
+      "description": "The creator of the collection."
+    },
+    "hasAccessRestrictions": {
+      "type": "boolean",
+      "description": "Indicates whether the collection has access restrictions."
+    },
+    "ods:collection_name": {
+      "type": "string",
+      "description": "The name of the collection."
+    },
+    "description": {
+      "type": "string",
+      "description": "A description of the collection."
+    },
+    "ods:collection_type": {
+      "type": "string",
+      "enum": ["Virtual Collection"]
+    },
+    "filters": {
+      "type": "array",
+      "description": "Filters selected for the collection (applicable to DVC)",
+      "items": {
+        "type": "object",
+        "properties": {
+          "filter_name": {
+            "type": "string",
+            "description": "Name or identifier/UUID for the filter"
+          },
+          "filter_value": {
```

Comment:
Couldn't this be multiple values. Let's say `dwc:country` could have Netherlands, Germany, Belgium. Would say to make this an array.